source/install/install-kubernetes.rst

```diff
@@ -71,23 +125,53 @@ Deploy Mattermost
         version: <VERSION_HERE>                       # Select a recent supported version of Mattermost. Example = 9.3.0
         licenseSecret: ""                             # If you created a license secret in step 1, put the secret name here
     
-  Save the file as ``mattermost-installation.yaml``. While recommended file names are provided, your naming conventions may differ. 
+  .. note::
+    While file names are provided, your file names can be different. The file names in this tutorial are used for clarity and organization.
 
-  Some of the most commonly-used fields include:
+  Here is a list of supported fields in the installation manifest file:
```

Comment:
It's great that you added more information on other available fields, but there are a lot more that aren't covered stillk so wondering if we want to keep the wording here closer the original to imply that the following list is incomplete.